docs/swagger.yaml

```
@@ -796,6 +850,32 @@ paths:
       summary: Gets a vehicle with uuid
       tags:
       - vehicle
+    put:
+      description: Will allow to update some wehicle data
+      parameters:
+      - description: Vehicle UUID
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)